shared/fixes/ansible/dconf_gnome_login_banner_text.yml

```diff
@@ -0,0 +1,50 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- (xccdf-var login_banner_text)
+- name: "@RULE_TITLE@"
+  file:
+    path: "/etc/dconf/db/{{ item }}"
+    owner: root
+    group: root
+    mode: 0755
+    state: directory
+  with_items:
+    - gdm.d
+    - gdm.d/locks
+ tags:
+   @ANSIBLE_TAGS@
```

Comment:
Thank you for your fast reply! However, you have introduced a small typo here - there is one space less here and the line below.

Reply:
Ah sorry about that.

Reply:
Sorry about not being more clear - the indentation of `tags:` and the line below (i.e. line 18 and 19 of the file) is one space less than it should be. The generated Ansible notebook is invalid because of that.

Reply:
Oh! I'm sorry about that. Ok fixed that.